test/class-properties.ts

```diff
@@ -1,4 +1,7 @@
+const enum TestResult { PASS, FAIL }
```

Comment:
This verifies that the `const enum` caveat no longer applies, too: https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats